pkg/loadbalancer/experimental/config.go

```diff
@@ -11,9 +11,10 @@ import (
 )
 
 type Config struct {
-	EnableExperimentalLB bool
-	RetryBackoffMin      time.Duration `mapstructure:"lb-retry-backoff-min"`
-	RetryBackoffMax      time.Duration `mapstructure:"lb-retry-backoff-max"`
+	EnableExperimentalLB          bool          `mapstructure:"enable-experimental-lb"`
+	RetryBackoffMin               time.Duration `mapstructure:"lb-retry-backoff-min"`
+	RetryBackoffMax               time.Duration `mapstructure:"lb-retry-backoff-max"`
+	TestFaultInjectionProbability float32       `mapstructure:"lb-test-fault-injection"`
```

Comment:
I'm quite against adding Configs only for testing purposes

Reply:
Not super familiar with the feature itself but would be nice if we could separate the testing code from the production code.

Reply:
I’ll split up this into a ”TestConfig” and make it an optional dependency and will only provide it in tests.